### ISSUE TITLE
New Backend Endpoint for GET/POST/PUT/DELETE static_page

### ISSUE THREAD

**Issue by jvillemare** [BITNULLS/storybook#344]
**Due Date:** Sprint S5

**Parent Issue:** #285

**Endpoint:** `GET, POST, PUT, DELETE /api/admin/static_page`

**Description:**
Create a new endpoint that allows an admin to get, create, update, or delete a `static_page`. A static page is a page that the admin can edit that will be displayed ultimately at `/static/page/<str:static_page_name>` that will host the website's about page, cookie page, etc.

**Objectives:**
 - [ ] Edit `edu_storybook/api/admin.py`, adding the endpoint.
 - [x] It is similar in structure to the `api/admin.py`'s book question endpoint that GET, POST, PUT, and DELETE (similar structure of the `if request.method == 'GET'` etc)
 - [ ] This should be easy, since GET is just a `SELECT`, a POST request is `INSERT` to the database, PUT is `UPDATE`, and DELETE is `DELETE` of two parameters
 - [x] GET just requires `static_page_name` as a string. Look up in database and return
 - [ ] POST is just `static_page_name` and `static_page_content`, where the `..._content` is the huge string of Markdown text, which will likely include new lines (`\n`)
 - [ ] PUT requires both of the same parameters as the POST
 - [ ] DELETE requires only the `static_page_name`
 - [ ] Create a new template `static_page.html`, add it to `templates.py`'s dict and class. For now, give it a super basic design (navbar pilcrow at the top, and pilcrow for the `content`. @/vivatheest can fix up the design of this page; make a new issue for that)
 - [ ] Create a new endpoint in `edu_storybook.api.index` called `api_get_static_page` with the route `/static/page/<str:static_page_name>`. Just run a simple select on the database, and then template in the static page content
 - [ ] Demonstrate a successful test, screenshot
 - [ ] Make a PR.

**Recommended Assignee:** Either @AmyFeld, @wagnerb3, or @bluehenraj 

**Blocked By:** #343
**Blocking Issue:** #362
